huaweicloud/services/vpc/resource_huaweicloud_vpc_subnet.go

```diff
@@ -213,15 +214,25 @@ func resourceVpcSubnetV1Create(d *schema.ResourceData, meta interface{}) error {
 
 }
 
-func resourceVpcSubnetV1Read(d *schema.ResourceData, meta interface{}) error {
-	config := meta.(*config.Config)
-	subnetClient, err := config.NetworkingV1Client(GetRegion(d, config))
+// GetVpcSubnetFromServer is a method to obtain subnet informations through subnet ID.
+func GetVpcSubnetFromServer(d *schema.ResourceData, config *config.Config, subentId string) (*subnets.Subnet, error) {
```

Comment:
suggest to define the method as `GetVpcSubnetByID`

Reply:
The function name has been updated.